src/pyfaf/actions/stats.py

```diff
@@ -120,7 +121,7 @@ def components(self, cmdline, db, opsys, release):
         for num, (comp, count, reports) in enumerate(results):
             if reports:
                 out += ("{0}. {1} seen {2} times ({3:.0%} of all reports)\n"
-                        .format(num + 1, comp, count, count / float(total_num)))
+                        .format(num + 1, comp, count, count // float(total_num)))
```

Comment:
I guess this should stay just /. Or you should change the format string.

Reply:
You are right, it should stay ```/``` because the second operator is ```float```.